src/ManifestTests/IntegrationTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using AccessibilityInsights.SetupLibrary;
+using AccessibilityInsights.VersionSwitcher;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+
+namespace ManifestTests
+{
+    [TestClass]
+    public class IntegrationTests
+    {
+        private static readonly string RawManifestFilePath = Path.GetFullPath(Path.Combine(@"..\..\..\..", @"bld\ReleaseInfo.json"));
+        private static readonly string SignedManifestFilePath = Path.GetFullPath(Path.Combine(@"..\..\..\..", @"Manifest\bin\Release\net48\AccessibilityInsights.Manifest.dll"));
+        private static readonly string MsiFilePath = Path.GetFullPath(Path.Combine(@"..\..\..\..", @"MSI\bin\Release\AccessibilityInsights.msi"));
+        private static readonly ChannelInfo rawInfo = FileHelpers.LoadDataFromJSON<ChannelInfo>(RawManifestFilePath);
+
+        [TestMethod]
+        public void MsiFileEists()
+        {
+            Assert.IsTrue(File.Exists(MsiFilePath), $"Can't find MSI file at {MsiFilePath}");
+        }
+
+        [TestMethod]
+        public void RawManifestFileExists()
+        {
+            Assert.IsTrue(File.Exists(RawManifestFilePath), $"Can't find raw manifest file at {RawManifestFilePath}");
+        }
+
+        [TestMethod]
+        public void EmbeddedManifestFileExists()
+        {
+            Assert.IsTrue(File.Exists(SignedManifestFilePath), $"Can't find signed manifest file at {SignedManifestFilePath}");
+        }
+
+        [TestMethod]
+        public void MsiFileSizeIsCorrectInRawManifest()
+        {
+            Assert.AreEqual(new FileInfo(MsiFilePath).Length, rawInfo.MsiSizeInBytes);
+        }
+
+        [TestMethod]
+        public void MsiSha512IsCorrectInRawManifest()
+        {
+            Assert.AreEqual(InstallationEngine.ComputeSha512(MsiFilePath), rawInfo.MsiSha512);
+        }
+
+        [TestMethod]
+        public void MinimumVersionIsNullInRawManifest()
+        {
+            Assert.IsNull(rawInfo.MinimumVersion);
+        }
+
+        [TestMethod]
+        public void EmbeddedManifestMatchesRawManifest()
+        {
+            using (Stream stream = new FileStream(SignedManifestFilePath, FileMode.Open, FileAccess.Read, FileShare.Read))
+            {
+                stream.Position = 0;
+
+                ChannelInfo signedInfo = ChannelInfoUtilities.GetChannelInfoFromSignedManifest(stream, (_) => true);
```

Comment:
> It would be cool if this bypass were skipped in the SignedRelease build, but I wasn't sure how to make that work.

I wonder if there's a way to use preprocessor directives to do this? We could conditionally define a `SIGNED_BUILD` constant using [DefineConstants](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/language#defineconstants) and use it to have an if/else check here along [these lines](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#conditional-compilation)?